cli/cli/commands/files/upload/upload.go

```diff
@@ -51,10 +52,11 @@ var FilesUploadCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisC
 			isEnclaveIdArgOptional,
 			isEnclaveIdArgGreedy,
 		),
-		{
-			Key:            pathArgKey,
-			ValidationFunc: validatePathArg,
-		},
+		file_system_path_arg.NewFilepathOrDirpathArg(
+			pathArgKey,
+			isPathArgOptional,
+			nil,
```

Comment:
can we swap the `nil` magic constant for a named constant per https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/main/general-style-guide.md#replace-magic-constants-with-constants-named-for-their-purpose-not-their-value so it's clear what the arg is?

Reply:
I added a function set to nil to be used for this use case.